src/moveroplot/station_scores.py

```diff
@@ -432,9 +509,9 @@ def _add_datapoints2(fig, data, score, ax, min, max, unit, param, debug=False):
     # Only modify ticks for the FBI case
     if score.startswith("FBI"):
         custom_ticks = (
-            [0.3, 0.5, 0.7, 0.9, 1, 2, 3]
+            [0.3, 0.5, 0.75, 1, 1.5, 2, 3]
             if param.startswith("CLCT")
-            else [0.1, 0.3, 0.5, 0.7, 0.9, 1, 2, 4, 6, 8, 10]
+            else [0.1, 0.3, 0.5, 0.75, 1, 1.5, 2, 4, 6, 8, 10]
         )
```

Comment:
Having such settings in the code is acceptable for now because of time constraints, but it is actually not nice that such settings are hardcoded in the code. It would be better if they could be at least in a separate file. A perfect although laborious solution would be to separate all such settings from the code and put them into a config file, which is read in at the start of the program.